<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpConnectionWidget</class>
 <widget class="QWidget" name="UdpConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Udp</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,0">
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Адрес:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Исходящий порт:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="listenPort">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>10100</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="autoReconnect">
     <property name="text">
      <string>Пытаться восстановить соединение</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Входящий порт:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="sendPort">
     <property name="text">
      <string>10000</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="host">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="autoSendPort">
     <property name="text">
      <string>Авто</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="autoListenPort">
     <property name="text">
      <string>Авто</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>host</tabstop>
  <tabstop>sendPort</tabstop>
  <tabstop>autoSendPort</tabstop>
  <tabstop>listenPort</tabstop>
  <tabstop>autoListenPort</tabstop>
  <tabstop>autoReconnect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
